### ISSUE TITLE
"403:Unknown client" errors in 311-indexer (salesforce change manager 000127)

### ISSUE THREAD

**Issue by fionawhim** [CityOfBoston/digital#175]
We’re getting errors from the Salesforce streaming API every 5–20 minutes about an "unknown client". We're currently restarting so things work, but this indicates there may be a deeper problem that should be resolved.

```
{
    "clientId": "2st8w3gricczecj1a6kdr718go6m",
    "advice": {
        "interval": 0,
        "reconnect": "none"
    },
    "channel": "/meta/connect",
    "id": "8",
    "error": "403::Unknown client",
    "successful": false
}
```

**Comment by fionawhim** [comment]
The new OAuth app for the indexer seems ot have addressed the frequency of these.

Still blocked on getting an app for devint2, and then we can close this.

**Comment by fionawhim** [comment]
This actually is still a problem in UAT.

I sent an email out to Jai and Nik asking about what the escalation steps are to get it resolved.

**Comment by fionawhim** [comment]
This is still an issue. Pinging the email.

**Comment by reillyzlab** [comment]
Not ideal but can be resolved post go live as noted in salesforce

**Comment by reillyzlab** [comment]
closing because no longer relevant
